src/srtctl/backends/sglang.py

```
@@ -18,22 +18,26 @@
 from srtctl.core.config import get_srtslurm_setting
 from srtctl.core.sweep import expand_template
 
-from .base import Backend
 
-
-class SGLangBackend(Backend):
+class SGLangBackend:
     """SGLang backend for distributed serving."""
 
     def __init__(self, config: dict, setup_script: str = None):
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)